sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -59,10 +60,8 @@
 """
```

Comment:
Will be interesting to see if there is a way to remove `text` completely from the GROUP BY clause, since that causes an execution of a hash function over the entire text for each row.

Reply:
If sql_handle maps 1:1 with sql query text, I think it should be possible to do another CTE `select TOP 1 text from qstats 
 cross apply dm_exec_sql_text(qstats.sql_handle) group by sql_handle` and then join it with the CTE above on sql_handle.